### DISCUSSION TITLE
500 error when paging the API

### DISCUSSION THREAD

**Discussion by atvaccaro** [transitland/transitland#358] [Q&A] [answered]
Hey all!

We've been paging the v2 feeds endpoint at https://github.com/cal-itp so we can identify feeds that we know about but don't show up in aggregators. Just in the last couple of days, our job starting getting 500 errors after several pages.

I was able to reproduce this via Postman locally (`apikey` removed)
`https://transit.land/api/v2/rest/feeds?limit=100&after=1241` => 200
`https://transit.land/api/v2/rest/feeds?limit=100&after=1242` => 500 with `request error` content

The response metadata does point at further pages, so our code attempted to continue paging.

```
"meta": {
        "after": 1352,
        "next": "https://api.transit.land/api/v2/rest/feeds?after=1352&limit=100"
    }
```

**Comment by irees** [comment, marked as answer]
Hi - thanks for identifying this problem. It's the result of a recent code change and I have a fix that should be implemented shortly.

**Reply by irees** [reply]
hi @atvaccaro  - this should be resolved now, can you check? I had a typo in an enum 😭  https://github.com/interline-io/transitland-server/pull/61

**Reply by atvaccaro** [reply]
Yes, looks good on our end! Thank you so much for the super fast response. 😄
